homeassistant/components/motioneye/switch.py

```diff
@@ -19,18 +19,17 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
-from homeassistant.util import slugify
 
 from . import MotionEyeEntity, listen_for_new_cameras
 from .const import CONF_CLIENT, CONF_COORDINATOR, DOMAIN, TYPE_MOTIONEYE_SWITCH_BASE
 
 MOTIONEYE_SWITCHES = [
-    KEY_MOTION_DETECTION,
-    KEY_TEXT_OVERLAY,
-    KEY_VIDEO_STREAMING,
-    KEY_STILL_IMAGES,
-    KEY_MOVIES,
-    KEY_UPLOAD_ENABLED,
+    (KEY_MOTION_DETECTION, "Motion Detection", True),
```

Comment:
Keep an eye on #53357. It will make it easy to streamline this list once it's merged.